### DISCUSSION TITLE
Expo Go crashes when trying to load my app

### DISCUSSION THREAD

**Discussion by captainaraf** [expo/expo#17568] [Archived: Questions]
Some days ago I have created a react native mobile app with expo, that worked totally fine. I could open in emulator and as well as on my phone. But, now I am trying to create another app. This app was also working perfectly fine, but 2 or 3 days ago when I wanted to open the app with expo go on my phone, expo go closed after building the app. I cannot even open it in an emulator, it shows an error "adb is not recognized as internal or external bash command". I have tried recreating a project, doesn't help.

**Comment by raajnadar** [comment]
This is related to your PC config, looks like due to improper env config.
